examples/threathunter/threat_intelligence/threatintel.py

```diff
@@ -62,11 +64,12 @@ def push_to_cb(self, feed_id, results):
 
         log.debug("Sending results to Carbon Black Cloud.")
 
-        try:
-            feed.append_reports(report_list_to_send)
-            log.info(f"Appended {len(report_list_to_send)} reports to ThreatHunter Feed {feed_id}")
-        except Exception as e:
-            log.debug(f"Failed sending {len(report_list_to_send)} reports: {e}")
+        if report_list_to_send:
```

Comment:
Asking for clarification: Does this fail if the list is emtpy?

Reply:
If the list is empty, and we didn't have the check for an empty list, there would be an attempt to append 0 reports, which succeeds but isn't necessary

Reply:
This seems reasonable, whatever you think is the best User Experience as you have the most knowledge of the use case